diff --git a/compose/project.py b/compose/project.py
index 696c8b04..32ddf556 100644
--- a/compose/project.py
+++ b/compose/project.py
@@ -4,6 +4,7 @@ from __future__ import unicode_literals
 import datetime
 import logging
 import operator
+import os
 import re
 from functools import reduce
 from os import path
@@ -103,7 +104,7 @@ class Project(object):
 
         for service_dict in config_data.services:
             service_dict = dict(service_dict)
-            if use_networking:
+            if use_networking and 'NO_DOCKER_SERVICE' not in os.environ:
                 service_networks = get_networks(service_dict, networks)
             else:
                 service_networks = {}
